centrifuge/apps
# Add solver setup

### Description



This pull request... adds solver to the UI. Currently, the solver can only provide solutions if the available reserve is too low to execute all order.

To test:
1. start with a pool at `Pool Value === 0`
2. set max reserve to 10'000
3. invest 6000 in junior tranche
4. invest 6000 in senior tranche
5. close epoch --> will go into submission period
6. submit solution and wait for challenge time to pass (will take 10 blocks)
7. execute the epoch after challenge time passes. If successful, the pool value should be the same as the max reserve.



- #883 


### Approvals



- [ ] Designer

### Screenshots


<img width="877" alt="close-epoch" src="https://user-images.githubusercontent.com/24418964/183681651-8234d2a4-4c53-463e-bf29-5ddcbc086155.png">
<img width="879" alt="submission-period" src="https://user-images.githubusercontent.com/24418964/183681724-bfc4dd7b-56b7-475c-b717-caec70c34328.png">
<img width="879" alt="challenge-period" src="https://user-images.githubusercontent.com/24418964/183681763-54d6acdc-e33c-49aa-af01-f761cb7ccc66.png">
<img width="878" alt="execution-period" src="https://user-images.githubusercontent.com/24418964/183681810-995feb61-e096-461c-ba94-39bccf20cce6.png">


### Impact
